interfaces/fiveg_n2/v0/schema.py

```diff
@@ -0,0 +1,32 @@
+"""This file defines the schemas for the provider and requirer sides of the `fiveg_n2` interface.
+It exposes two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {"n2_ip_address": "192.168.251.6"}
+    RequirerSchema:
+        unit: <empty>
+        app:  <empty>
+"""
+
+from pydantic import BaseModel, IPvAnyAddress, Field
+
+from interface_tester.schema_base import DataBagSchema
+
+
+class MyProviderAppData(BaseModel):
+    n2_ip_address: IPvAnyAddress = Field(
```

Comment:
I think `amf_ip_address` makes more sense. The n2 interface is used by both the AMF and gNodeB and using `n2` makes it unclear which ip it is.

Besides, are we sure that IP is the correct attribute to use? From the gNodeB it seems like it can be expecting a hostname.